pages.la/common/tldr.md

```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> ສະແດງຕົວຢ່າງແບບງ່າຍສຳລັບເຄື່ອງມືເທິງ command-line ຈາກໂປຣເຈັກ tldr-pages.
+> ຂໍ້ມູນເພີ່ມເຕີມ: <https://tldr.sh>.
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
+
+`tldr {{command}}`
```

Comment:
Translate `{{command}}` and `{{subcommand}}` command arguments in this page. 

Also, add "Print the tldr page for a specific subcommand:" and it's command from our [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr.md).